try to display correct mv note under "warning" row

Philotto opened the pull request.

Philotto committed:
try to display correct mv note under "warning" row

Mobe1969 commented:
I'll just update this in mine - your fine name is not quite correct either. Eg 
Movie BEQs/Snake Eyes- G.I. Joe Origins (2021)(It) DD 5.1.xml
So missing a space after (It)

Also, is that old DD or DD+? Eg AC3 vs EAC3?

Philotto committed:
update name with space after (It)

i can confirm is DD , Dolby Digital Audio             Italian         640 kbps        5.1 / 48 kHz / 640 kbps
here in italy is a true issues the audio quality on movie. 
I can tell you that we have no more than 10 movies in italian in Atmos, for example, maybe no more than 5.
we are happy when see dts hdma

Philotto committed:
Merge branch 'master' into master

Philotto commented:
I don't know if now it is ok, regarding Dolby Digital yes, it is correct,
we have the old Dolby digital (a real problem here in Italy the audio
quality delivered with movies)

Il ven 19 nov 2021, 02:30 Mobe1969 ***@***.***> ha scritto:

> I'll just update this in mine - your fine name is not quite correct
> either. Eg
> Movie BEQs/Snake Eyes- G.I. Joe Origins (2021)(It) DD 5.1.xml
> So missing a space after (It)
>
> Also, is that old DD or DD+? Eg AC3 vs EAC3?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Mobe1969/miniDSPBEQ/pull/12#issuecomment-973645229>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV7QRZJJRDUU5LO2IHMYJALUMWSB5ANCNFSM5IKNEJKQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
